test/e2e/specs/editor/blocks/image.spec.js

```diff
@@ -739,10 +742,90 @@ test.describe( 'Image', () => {
 			new RegExp( filename )
 		);
 	} );
+
+	test( 'image inserted via link should appear in the frontend published post content', async ( {
+		editor,
+		page,
+	} ) => {
+		await editor.insertBlock( { name: 'core/image' } );
+		const imageBlock = editor.canvas.locator(
+			'role=document[name="Block: Image"i]'
+		);
+		await expect( imageBlock ).toBeVisible();
+
+		await imageBlock
+			.getByRole( 'button' )
+			.filter( { hasText: 'Insert from URL' } )
+			.click();
+
+		const form = page.locator(
+			'.block-editor-media-placeholder__url-input-form'
```

Comment:
Seems like there should be a `dialog` here but it doesn't exist 😅 . I think it's okay to use class names as long as we add a comment above it 👍

Reply:
Ok got it! Will add a comment 👍

Reply:
Comment added.